docs/api/commands.md

```diff
@@ -40,6 +40,9 @@ looks for an `index.js`.
 building the app. Assumes your app is already running.
 * `-d, --dev`: Start the test server and keep it running until manually killed.
 For use with hot-reloading.
+* `--xml`: Write the test results to `cavy_results.xml`, which
```

Comment:
You can already run your Cavy tests in CI tools (like we do in Circle CI) so maybe just...

"Write the test results to `cavy_results.xml`, conforming to the JUnit XML specification"